<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seidel</class>
 <widget class="QWidget" name="Seidel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tabla_solve">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>631</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultado">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_resoult">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bt_funcion">
      <property name="text">
       <string>Ingresar Valores</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>Tolerancia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>niteraciones</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_solve">
      <property name="text">
       <string>Solucionar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tabla_A">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_b">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>161</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_X">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>40</y>
     <width>161</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matriz A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Matriz b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>verctor X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
